doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,51 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra                                      > NEW LINE
+
+sudo kdb mount hello.js /hello json
+kdb set user/hello/hello/world "Hello World"
+kdb get user/hello/hello/world
+kdb file user/hello/hello/world
+vi /home/elektra/.config/hello.js
+# Edit file to say "Hello World from File!"
+kdb get user/hello/hello/world
+kdb editor user/hello ini
+# Edit file to say "Hello World from INI!"
+kdb get user/hello/hello/world
+echo "Elektra can export to various formats!"
+kdb export user/hello
+kdb export user/hello tcl
+kdb export user/hello yaml                                      > NEW LINE
+echo "Remove entries and file"
+kdb rm user/hello/hello/world
+cat /home/elektra/.config/hello.ini
+sudo kdb umount /hello
+echo "Now let us change something.. With validation"
+vi specification.ini                                       > NEW START
+# content:
+# []
+# mountpoint = specification.ini
+# [hostip]
+# check/ipaddr =
+# [timeout]
+# check/range = 0-100
```

Comment:
This should be looked up in METADATA.ini (`kdb mount-info` and then `kdb meta-ls system/info/elektra/metadata/#0/check/ipaddr`)

Reply:
for that I would need to mount metadata.ini ... 
when i run `sudo kdb mount-info` I get no result in docker
apart from that it is a tutorial and IMO too much information anyway so it is probably better to leave this part to tutorial reading

Reply:
yes, then put this to the tutorial. @mpranj any idea why `mount-info` cannot find METADATA.ini? #1663

Reply:
I think it does work. I get the same result on my host machine and in docker:

```
elektra $ sudo kdb mount-info
elektra $ kdb meta-ls system/info/elektra/metadata/#0/check/ipaddr
comments
comments/#0
internal/ini/key/last
internal/ini/order
internal/ini/parent
internal/ini/section
```

Reply:
Thank you for trying!

Reply:
Ok, this command is very confusing so I just want to add my opinion to this command which others might also think of.
1. there is no man page where I could read what it does
2. in the name is "info" which means for me that it just outputs something and does not change something (it actually mounts). A better name would be mount-defaults or something like that
3. The command does not give any output at all, not even success or what it actually has done. It should at least say it mounted the files. The "info" is especially confusing when it outputs nothing

Reply:
I agree with your suggestions. You are in the best position to improve the situation.

Reply:
Sorry but I do not have time for this